website/docs/spec/index.md

```diff
@@ -111,12 +111,14 @@ All MCAP records are serialized as follows:
 
 Record type is a single byte opcode, and record content length is a uint64 value.
 
-Records may be extended by adding new fields at the end of existing fields. Readers should ignore any unknown fields.
+Future changes to this specification may extend records by adding new fields at the end of existing fields. Readers should ignore any unknown fields.
```

Comment:
if we are introducing this language it would be good to do in a separate PR, and ensure we have conformance tests for readers well ahead of introducing any new fields.